src/audio/host-zephyr.c

```diff
@@ -244,8 +244,6 @@ void host_common_update(struct host_data *hd, struct comp_dev *dev, uint32_t byt
 	struct comp_buffer __sparse_cache *source;
```

Comment:
@btian1 git commit is best as that is stored in history, and anyone looking at the code later on, can find the reason why the change was made from git history. This is good to spread good coding practises, avoid people from removing useful code (if they don't understand why it is there) and give context in which conditions the savings are realized. This can be very short though, just key bits (what are the measure units, what platform and what was measured).